### ISSUE TITLE
Typescript declaration file?

### ISSUE THREAD

**Issue by imMohika** [feat-agency/vite-plugin-webfont-dl#1]
## Error:

```
Could not find a declaration file for module 'vite-plugin-webfont-dl'. 'C:/.../node_modules/.pnpm/vite-plugin-webfont-dl@1.0.7_vite@2.7.13/node_modules/vite-plugin-webfont-dl/dist/vite-plugin-webfont-dl.js' implicitly has an 'any' type.
Try `npm i --save-dev @types/vite-plugin-webfont-dl` if it exists or add a new declaration (.d.ts) file containing `declare module 'vite-plugin-webfont-dl';
```

###  `vite.config.ts`

```
import { UserConfig } from "vite";
...
import { ViteWebfontDownload } from "vite-plugin-webfont-dl";
const config: UserConfig = {
...
    ViteWebfontDownload([
      "https://fonts.googleapis.com/css2?family=Raleway&display=swap",
      "https://fonts.googleapis.com/css2?family=Open+Sans&display=swap",
    ]),
...
};
export default config;
```

**Comment by 0xb4lint** [comment]
I've just released **[v1.0.9](https://github.com/feat-agency/vite-plugin-webfont-dl/releases/tag/v1.0.9)**, this version contains an `index.d.ts` declaration file.

Could you please check it?

**Comment by imMohika** [comment]
FIXED

thanks <3
